### PR TITLE
Update WC trace to show filter class name

### LINKED ISSUE #18578: Update trace to show filter class info
Currently, service team has to  use an -Xtrace to find out a filter's class when debugging field issues.

For example:

[9/16/21, 11:07:40:835 EDT] 00000035 filter        1 com.ibm.ws.webcontainer.filter.WebAppFilterChain doFilter executing filter [Test Filter] request [com.ibm.ws.webcontainer40.srt.SRTServletRequest40@6624be83] response [com.ibm.ws.webcontainer40.srt.SRTServletResponse40@69137966]

showing the filter name but not the filter class name.

This issue updates to show the filter name along with filter class name when full WC trace is enabled.  Example,

[9/16/21, 10:50:39:131 EDT] 00000038 filter        1 com.ibm.ws.webcontainer.filter.WebAppFilterChain doFilter enter filter [Test Filter] **class [com.ibm.wl3eng.servlets.TestFilter@166ceef8]** request [com.ibm.ws.webcontainer31.srt.SRTServletRequest31@9abc5bc1] response [com.ibm.ws.webcontainer31.srt.SRTServletResponse31@e1f05d07]

### PR DESCRIPTION
**Pull request by pmd1nh**
fixes #18578

### CODE REVIEW

**Comment by pmd1nh** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer.servlet.3.1_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer_fat_part2,com.ibm.ws.webcontainer_fat_servlet31toleration,com.ibm.ws.webcontainer_fat_servlet31toleration_part2,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0_part2,com.ibm.ws.webcontainer_fat_lWAS,com.ibm.ws.webcontainer_fat_servlet31_webcontainerlWAS,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0.lWAS

File: dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/filter/WebAppFilterChain.java [hunk 1]
```diff
@@ -126,66 +130,79 @@ else if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled() && logger.isLoggab
...
-                    }
-                    HttpInboundConnection httpInboundConnection = null;
-                    if (request instanceof IExtendedRequest) {
-                    	IExtendedRequest srtReq = (IExtendedRequest)request;
-                        IRequestExtended iReq = (IRequestExtended)srtReq.getIRequest();
-                        if (iReq != null) {
-                            httpInboundConnection = iReq.getHttpInboundConnection();
-                            logger.logp(Level.FINE, CLASS_NAME, "invokeTarget", "HttpInboundConnection: " + httpInboundConnection);
+        if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.FINE)){  //306998.15
+            logger.entering(CLASS_NAME, "invokeTarget", "request [" + request + "] response [" + response + "]");
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/filter/WebAppFilterChain.java:144]: What is` 306998.15`? I assume just a copy paste thing? Can we remove that?

File: dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/filter/WebAppFilterChain.java [hunk 2]
```diff
@@ -126,66 +130,79 @@ else if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled() && logger.isLoggab
...
+
+        }
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/filter/WebAppFilterChain.java:145]: Remove this extra new line.

**Comment by pmd1nh** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer.servlet.3.1_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer_fat_part2,com.ibm.ws.webcontainer_fat_servlet31toleration,com.ibm.ws.webcontainer_fat_servlet31toleration_part2,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0_part2,com.ibm.ws.webcontainer_fat_lWAS,com.ibm.ws.webcontainer_fat_servlet31_webcontainerlWAS,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0.lWAS

**Review by pnicolucci — APPROVED** [review]

**Comment by pmd1nh** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer.servlet.3.1_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer_fat_part2,com.ibm.ws.webcontainer_fat_servlet31toleration,com.ibm.ws.webcontainer_fat_servlet31toleration_part2,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0_part2,com.ibm.ws.webcontainer_fat_lWAS,com.ibm.ws.webcontainer_fat_servlet31_webcontainerlWAS,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0.lWAS

**Comment by pmd1nh** [comment]
all WC buckets passed.  Not related failed tests:



com.ibm.ws.collective.routing.member_fat FAT tests | 17 | 1 | 0 |   |  
-- | -- | -- | -- | -- | --
com.ibm.ws.ui.tool.explore_fat.gui.nd FAT tests | 48 | 4 | 0 |   |  
getImage | 1 | 1 | 0 |   |

**Comment by pmd1nh** [comment]
#run-libby-bot

**Pull request merged**